replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+
+dat_egood$music <- as.numeric(dat_egood$music)
+dat_egood <- dat_egood %>%
+  mutate(music.recode2 = (music - 1))
+View(dat_egood)
+
+#Set factor names:
+Music <- factor(dat_egood$music.recode2,
+                    levels = c(0, 1),
+                    labels = c("Liked", "DisLiked"))
+
+ChosenPen <- factor(dat_egood$chose_advertised_pen,
+                levels = c(1, 0),
+                labels = c("ChoseAdPen", "NotChoseAdPen"))
+
+#Compare counts of each gender in paper to data for coding:
+count(dat_egood, geslacht1)
+## 1 = male
+## 2 = female
+
+# need to check which is which:
+Gender <- factor(dat_egood$geslacht1,
+                    levels = c(1,2),
+                    labels = c("Male", "Female"))
+
+# for relationship between advertised pen and music, music 2 (disliked) seems significant:
+fit1 <- glm(ChosenPen ~ Music, data = dat_egood, family = binomial)
+fit1 <- tidy(fit1)
+fit1
+
+#### calculate odds 
+fit1$estimate
+fit1$estimate[1] + fit1$estimate[2] ## 1.216
+## odds ratio
+exp(fit1$estimate[2]) ##4.43
+## predicted probabilities:
+# Disliked Music:
+(exp(fit1$estimate[1] + fit1$estimate[2])) / (1 + exp(fit1$estimate[1] + fit1$estimate[2]))
+# Liked Music: 
+exp(fit1$estimate[1]) / (1 + exp(fit1$estimate[1])) 
+
+# does the relationship change when looking ONLY at males & ONLY females?
+
+male.dat_egood <- dat_egood %>%
+  filter(geslacht1 == "1")
+#count(male.dat_egood, geslacht1)
+
+female.dat_egood <- dat_egood %>%
+  filter(geslacht1 == "2")
+#count(female.dat_egood, geslacht1)
+
+# MALES:
+fitmale <- glm(ChosenPen ~ Music, data = male.dat_egood, family=binomial)
+fitmale <- tidy(fitmale)
+fitmale
+#### calculate odds 
+fitmale$estimate
+fitmale$estimate[1] + fitmale$estimate[2] ## 1.216
+## odds ratio
+exp(fitmale$estimate[2]) ##4.43
+
+# FEMALES:
+fitfemale <- glm(ChosenPen ~ Music, data = female.dat_egood, family=binomial)
+fitfemale <- tidy(fitfemale)
+fitfemale
+#### calculate odds 
+fitfemale$estimate
+fitfemale$estimate[1] + fitfemale$estimate[2] ## 1.216
+## odds ratio
+exp(fitfemale$estimate[2]) ## 4.43
+
+### predicted probabilities = confirmed, gender does not have a moderating effect on the likelihood 
+# of choosing the advertised pen when listening to liked versus disliked music.
+# Males - Disliked Music = 0.77
+(exp(fitmale$estimate[1] + fitmale$estimate[2])) / 
+  (1 + exp(fitmale$estimate[1] + fitmale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitmale$estimate[1]) / (1 + exp(fitmale$estimate[1])) 
+
+# Females - Disliked Music = 0.77
+(exp(fitfemale$estimate[1] + fitfemale$estimate[2])) / 
+  (1 + exp(fitfemale$estimate[1] + fitfemale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitfemale$estimate[1]) / (1 + exp(fitfemale$estimate[1])) 
+
+
+
+```
+Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the table. Further, the "eval = FALSE" parameter was added to the top code chunk to prevent printing of the code.
```

Comment:
The extension was a really great idea! I really feel like you took this analysis further in the sense that you tried to test an interesting and meaningful hypothesis. It would be better if your files had clearer descriptions and they was more description of specifically what you were trying to replicate. It would also have been useful to compare your replication to what the authors had done.